src/jmclient/yieldgenerator.py

```diff
@@ -268,6 +269,9 @@ def select_output_address(self, input_mixdepth, offer, amount):
         cjoutmix = (input_mixdepth + 1) % (self.wallet_service.mixdepth + 1)
         return self.wallet_service.get_internal_addr(cjoutmix)
 
+    def select_change_address(self, input_mixdepth, change_amount):
```

Comment:
We try to use type hints in all new code.

```suggestion
    def select_change_address(self, input_mixdepth: int, change_amount: int) -> str:
```

Reply:
I added a doc string too.